Multisig/UI/ENS/EnterENSNameViewModel.swift

```diff
@@ -28,34 +28,44 @@ class EnterENSNameViewModel: ObservableObject {
 
     private var subscribers = Set<AnyCancellable>()
 
-    func resolve(name: String) {
-        subscribers.forEach { $0.cancel() }
-
+    init() {
         $text
-            .map { v -> String in
-                self.reset()
+            .map { [weak self] v -> String in
+                self?.reset()
```

Comment:
In the other PR you used "guard let `self` = self else { return }" validation, I think it's safer if we have same validation here

Reply:
generally, if there would be more than 1 statement that uses `self`, that would be a valid concern, but for a single statement it doesn't matter. 

This is not future-proof though, as adding more calls to self will have to use the optional chaining. I'll update it.